.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
       - master
 jobs:
   clang-format-check:
-    runs-on: macos-11
+    runs-on: macos-10.15
```

Comment:
Any reason for changing this to macos-10.15?

Reply:
I changed it as I was not sure what was causing the tests to fail, now I know it was just the flakiness. I can change it to macos 11 if needed

Reply:
Changed it to "latest" for a mix of "latest" and an older version